packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts

```diff
@@ -19,20 +19,20 @@ export const getOkmsCredentialsQueryKey = (okmsId: string) => [
  *  Get okms Credential
  */
 
-export const getOkmsCredentialQueryKey = ({
+export const getOkmsCredential = async ({
   okmsId,
   credentialId,
 }: {
   okmsId: string;
   credentialId: string;
-}) => [`get/okms/resource/${okmsId}/credential/${credentialId}`];
+}): Promise<ApiResponse<OkmsCredential>> => {
+  return apiClient.v2.get(`okms/resource/${okmsId}/credential/${credentialId}`);
+};
 
-export const getOkmsCredential = async ({
+export const getOkmsCredentialQueryKey = ({
   okmsId,
   credentialId,
 }: {
   okmsId: string;
   credentialId: string;
-}): Promise<{ data: OkmsCredential }> => {
-  return apiClient.v2.get(`okms/resource/${okmsId}/credential/${credentialId}`);
-};
+}) => [`get/okms/resource/${okmsId}/credential/${credentialId}`];
```

Comment:
with the new structure folder, `queryKeys` are declared inside the used hook now